### DISCUSSION TITLE
HIOS uneccessarily expands size of the files, make arrays or remove altogether

### DISCUSSION THREAD

**Discussion by jgsmithHZN** [CMSgov/price-transparency-guide#232] [Q&A] [answered]
I am not sure about other plans, but we have rates that are the SAME across all of HIOS related Plans.   
Because the schema / requirements indicates push the use of HIOS code instead of our EIN for those products / plans.... we are forced to send out the same rates for every HIOS product...with all the network and provide info... Replicated  50 times (the number of HIOS products we have).   

I don't think that was the intended outcome (with respect to any paperwork reduction acts). 

It seems someone in creating the original schema, thought that different HIOS products drove different negotiated provider rates.  It doesn't for our company and I don't believe it does for others across the country.   

**Recommendation1:** Make PLAN names and/or plan IDs enabled as arrays as the NPIs and POS fields have been.   
_Rational_:  It will vastly reduce the size of the files we need to create. 

**Recommendation2**: Remove HIOS from the landscape altogether.  
_Rationale:_ ACA (HIOS) related products were required to register with HIOS only up until 2019.  
Then the HIOS registration was turned off.  Therefore, all HIOS ids are effectively going stale every year as products are replace with new products.  According to this - the need to have these unique health plan identifiers was rescinded in 2019.   At least I _THINK_ it's referring to the same HIOS IDs... 
 [https://www.federalregister.gov/documents/2019/10/28/2019-23507/administrative-simplification-rescinding-the-adoption-of-the-standard-unique-health-plan-identifier#p-45](url).

**Comment by ghost** [comment, marked as answer]
Have you checked the recent schema changes? The updated schema has a <html>
<body>


reporting_plans | In-Network Plans | Array | An array of reporting plan object types
-- | -- | -- | --



</body>
</html>

that consists of 

<html>
<body>
<h4 style="box-sizing: border-box; margin-top: 24px; margin-bottom: 16px; font-size: 16px; font-weight: 600; line-height: 1.25; color: rgb(201, 209, 217); font-family: -apple-system, BlinkMacSystemFont, &quot;Segoe UI&quot;, Helvetica, Arial, sans-serif, &quot;Apple Color Emoji&quot;, &quot;Segoe UI Emoji&quot;; font-style: normal; font-variant-ligatures: normal; font-variant-caps: normal; letter-spacing: normal; orphans: 2; text-align: start; text-indent: 0px; text-transform: none; white-space: normal; widows: 2; word-spacing: 0px; -webkit-text-stroke-width: 0px; background-color: rgb(13, 17, 23); text-decoration-thickness: initial; text-decoration-style: initial; text-decoration-color: initial;">Reporting Plans Object</h4><p style="box-sizing: border-box; margin-top: 0px; margin-bottom: 16px; color: rgb(201, 209, 217); font-family: -apple-system, BlinkMacSystemFont, &quot;Segoe UI&quot;, Helvetica, Arial, sans-serif, &quot;Apple Color Emoji&quot;, &quot;Segoe UI Emoji&quot;; font-size: 16px; font-style: normal; font-variant-ligatures: normal; font-variant-caps: normal; font-weight: 400; letter-spacing: normal; orphans: 2; text-align: start; text-indent: 0px; text-transform: none; white-space: normal; widows: 2; word-spacing: 0px; -webkit-text-stroke-width: 0px; background-color: rgb(13, 17, 23); text-decoration-thickness: initial; text-decoration-style: initial; text-decoration-color: initial;">Information about the plan that is being reported on for the in-network negotiated rates for all items and services. Multiple<span> </span><code style="box-sizing: border-box; font-family: ui-monospace, SFMono-Regular, &quot;SF Mono&quot;, Menlo, Consolas, &quot;Liberation Mono&quot;, monospace; font-size: 13.6px; padding: 0.2em 0.4em; margin: 0px; background-color: var(--color-neutral-muted); border-radius: 6px;">Reporting Plan Objects</code><span> </span>can be included in this array for all of the plans that have identical negotiated rates in the<span> </span><code style="box-sizing: border-box; font-family: ui-monospace, SFMono-Regular, &quot;SF Mono&quot;, Menlo, Consolas, &quot;Liberation Mono&quot;, monospace; font-size: 13.6px; padding: 0.2em 0.4em; margin: 0px; background-color: var(--color-neutral-muted); border-radius: 6px;">in_network</code><span> </span>array.</p>

Field | Name | Type | Definition | Required
-- | -- | -- | -- | --
plan_name | Plan Name | String | The plan name and name of plan sponsor and/or insurance company. | Yes
plan_id_type | Plan Id Type | String | Allowed values: "EIN" and "HIOS" | Yes
plan_id | Plan ID | String | The 14-digit Health Insurance Oversight System (HIOS) identifier, or, if the 14-digit HIOS identifier is not available, the 5-digit HIOS identifier, or if no HIOS identifier is available, the Employer Identification Number (EIN)for each plan or coverage offered by a plan or issuer. | Yes
plan_market_type | Market Type | String | Allowed values: "group" and "individual" | Yes


</body>
</html>

Your "we are forced to send out the same rates for every HIOS product...with all the network and provide info... Replicated 50 times (the number of HIOS products we have)." should work within the new schema as a 50 array element for a single file avoiding the duplication.

Welcome to the party!

**Reply by shaselton-usds** [reply]
Thanks @BobSyracuse!

@jgsmithHZN, thanks for the question. @BobSyracuse actually brought this up in the past. You can see an example implementation of combining multiple plans with the same rates/providers/items/services here: https://github.com/CMSgov/price-transparency-guide/blob/master/examples/in-network-rates/in-network-rates-fee-for-service-sample.json#L4-L14 In the example, there are two plans that are reporting for the single file (medicare and medicaid is obviously just sample data, but you get the picture).
